src/app/modals/firsttime/confirmpassphrase/confirmpassphrase.component.spec.ts

```diff
@@ -1,19 +1,37 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+
+import { ElectronService } from 'ngx-electron';
+import { PeerService } from '../../../core/rpc/peer.service';
```

Comment:
please remove all these imports and all the ones you added in the other component.spec.ts files